canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -96,6 +96,14 @@ export default class PropertiesController {
 		return this.commandStack;
 	}
 
+	setInitialEditorSize(initialEditorSize) {
```

Comment:
Thinking about this more this probably makes more sense to rename to setEditorSize and getEditorSize.  Sorry about that.